op-dispute-mon/cmd/main_test.go

```diff
@@ -297,7 +304,7 @@ func addRequiredArgsExcept(name string, optionalArgs ...string) []string {
 func requiredArgs() map[string]string {
 	args := map[string]string{
 		"--l1-eth-rpc":           l1EthRpc,
-		"--rollup-rpc":           rollupRpc,
+		"--rollup-rpc":           rollupRpc[0],
```

Comment:
```suggestion
		"--rollup-rpc":           strings.Join(rollupRpc, ","),
```
Just so we don't get surprised if the default value ever gets changed to have multiple urls.